test/regfox/regfox_api.integration.js

```diff
@@ -87,5 +87,14 @@ describe('regfox_api (integration testing)', () => {
       expect(registrantInfo.name).to.be.equal(TEST_NAME);
       expect(`${registrantInfo.customerId}`).to.be.equal(TEST_CUSTOMER_ID);
     });
+
+    it.allowFail('marks a registrant as complete', async () => {
+      assert.exists(bearerToken);
+      const registrationInfo = await getRegistrationInfo('26564608', bearerToken);
```

Comment:
Same as the other PR, can this magic number be a const somewhere near the top?

Reply:
Sure!